### PR TITLE
Fiverr PR

### PR DESCRIPTION
**Pull request by noahliechti**

### CODE REVIEW

File: server/routes/auth.js [hunk 1]
```diff
@@ -1,29 +1,19 @@
...
 // When logout, redirect to client
 router.get("/logout", (req, res) => {
-  req.logout(); // TODO: why does this not delete the session in the browser/db?
+  req.logout();
+  req.session.destroy(); // TODO: why does this not delete the session in the browser/db? // now session will be deleted from db after logout
   res.redirect(CLIENT_HOME_PAGE_URL);
 });
 
-// auth with twitter
-router.get("/twitter", passport.authenticate("twitter", { session: false }));
```
Comment (noahliechti) [inline server/routes/auth.js:17]: isn't this router needed? my frontend calls `auth/twitter` when clicking on the button

File: server/server.js [hunk 2]
```diff
@@ -21,37 +21,31 @@ app.use(
...
 app.use(passport.session());
 
-app.use(
-  cors({
-    origin: "http://localhost:3000",
-    methods: "GET,HEAD,PUT,PATCH,POST,DELETE",
-    credentials: true, // allow session cookie from browser to pass through
-  })
-);
+app.use(cors());
```
Comment (noahliechti) [inline server/server.js:39]: isn't this a vulnerability that now everybody can call the server? I'm thinking about when this is in production

File: server/routes/auth.js [hunk 3]
```diff
@@ -1,29 +1,19 @@
 const router = require("express").Router();
 const passport = require("passport");
-const CLIENT_HOME_PAGE_URL = "http://localhost:3000";
+const CLIENT_HOME_PAGE_URL = "http://localhost:4000/";
```
Comment (noahliechti) [inline server/routes/auth.js:3]: I think this should be `localhost:3000` because the frontend runs on this port

File: server/package.json [hunk 4]
```diff
@@ -4,13 +4,14 @@
   "description": "",
   "scripts": {
     "dev": "nodemon server.js",
```
Comment (noahliechti) [inline server/package.json:6]: can you make that on dev a local mongo is started and the NODE_ENV environment variable is set to `development`?

File: server/server.js (hunk 2 above)
Comment (noahliechti) [reply server/server.js:39]: and now I get an error with cors because when 'Access-Control-Allow-Origin' is '*' credentials are not supported

Comment (uzairanjum) [reply server/server.js:39]: reverted

File: server/routes/auth.js (hunk 3 above)
Comment (uzairanjum) [reply server/routes/auth.js:3]: ok

**Pull request merged**